builtin/providers/ultradns/resource_ultradns_rdpool.go

```diff
@@ -0,0 +1,239 @@
+package ultradns
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/Ensighten/udnssdk"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceUltradnsRdpool() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceUltradnsRdpoolCreate,
+		Read:   resourceUltradnsRdpoolRead,
+		Update: resourceUltradnsRdpoolUpdate,
+		Delete: resourceUltradnsRdpoolDelete,
+
+		Schema: map[string]*schema.Schema{
+			// Required
+			"zone": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"order": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				// 0-255 char
+				// FIXED | RANDOM | ROUND_ROBIN
```

Comment:
a validation func would be awesome. Also, why no default? Isn't there a default in the UI?